vllm/sampling_params.py

```diff
@@ -184,6 +186,11 @@ def __init__(
 
         self._verify_args()
         if self.use_beam_search:
+            print_warning_once(
```

Comment:
i would keep printing this for every single request (can be disabled by env var VLLM_NO_DEPRECATION_WARNING)

Reply:
Makes sense. Added the env variable. PTAL!